redhat/firehol/firehol.spec

```diff
@@ -13,6 +13,7 @@ Source1:        firehol.service
 Source2:        fireqos.service
 %endif
 BuildArch:      noarch
+BuildRequires:  iprange
```

Comment:
Just wanted to highlight the Release line, where the 2 should go

Github doesn't seem to let me to leave a comment or suggestion for a line you haven't modified, hence the huge block...

Reply:
In fact, I had a block highlighted but in the published comment we see only your change.

Release is on line 3!